src/main/java/edu/oakland/helper/admin/LocationDataPoint.java

```diff
@@ -69,6 +69,10 @@ public LocalDateTime getTime() {
     return time;
   }
 
```

Comment:
I would make sure you add the `@Override notation and a Javadoc
```suggestion
  /**
   * Creates a print-friendly string of the LocationDataPoint data.
   *
   * @return The print friendly string of the LocationDataPoint data.
   */
  @Override
```